digiwf-apps/packages/apps/digiwf-tasklist/src/components/schema/VMultiUserInput.vue

```diff
@@ -3,10 +3,11 @@
     <v-autocomplete
       ref="autocompletion"
       v-model="selectedUsers"
+      :aria-label="!isReadonly() ? label + ' Name eingeben und mit Pfeiltaste runter durch die Liste navigieren' : ''"
```

Comment:
```suggestion
      :aria-label="!isReadonly() ? label + ' Name eingeben und mit Pfeiltaste-Runter durch die Liste navigieren' : ''"
```

Bin mir nicht ganz sicher was korrekt ist, aber denke eher so